apollo-router/src/uplink/license_enforcement.rs

```diff
@@ -394,6 +394,11 @@ impl LicenseEnforcementReport {
                 .path("$.preview_demand_control")
                 .name("Demand control plugin")
                 .build(),
+            ConfigurationRestriction::builder()
```

Comment:
This needs a related change in https://github.com/apollographql/router/blob/dev/apollo-router/src/uplink/testdata/restricted.router.yaml

Reply:
Thanks, I've updated the test and snapshot.